var/spack/repos/builtin/packages/mirdeep/package.py

```diff
@@ -31,9 +31,9 @@ class Mirdeep(Package):
        by analyzing sequenced RNAs."""
 
     homepage = "https://www.mdc-berlin.de/8551903/en/"
-    url      = "https://www.mdc-berlin.de/45995549/en/research/research_teams/systems_biology_of_gene_regulatory_elements/projects/miRDeep/mirdeep2_0_0_8.zip"
+    url      = "https://www.mdc-berlin.de/media/16193"
 
-    version('2.0.0.8', 'a707f7d7ad4a2975fb8b2e78c5bcf483')
+    version('2.0.0.8', 'a707f7d7ad4a2975fb8b2e78c5bcf483', extension='.zip')
```

Comment:
I would actually rename this package to `mirdeep2` and change the version number to 0.0.8. That's how it appears on their website.